### DISCUSSION TITLE
Who is using Ariakit?

### DISCUSSION THREAD

**Discussion by diegohaz** [ariakit/ariakit#1661] [General]
Hey! 👋

If you or your company are using Ariakit, please leave a comment! Include the website (if public), the repository (if public), a description of how you're using it, and any other relevant information.

Also, please, let us know if we can put your logo on our site or README.

Thank you! :)

**Comment by Saeris** [comment]
We're using Reakit at @CodeSandbox in many of our new components and in a lot of refactors to aid us in added keyboard accessibility to our product! Personally I'm a big fan of how easy it's been to use, it's really simple to create highly reusable components using reakit as a base, since it's no problem to add styling on top of it.

You're welcome to add us to your list! If you need any specific assets from us let me know.

**Comment by lxcid** [comment]
We at @taskade is integrating Reakit as well to help improve our accessibility (focus and keyboard). Its still a WIP and not live yet. Feel free to add us.

 Thanks for the amazing hard work.

**Comment by frontendwizard** [comment]
We just started to use reakit @usabilla for our new components. The clean API and accessibility solutions are really amazing.

You're welcome to add us to the list :)

**Comment by StorytellerCZ** [comment]
Just published a new version of my project: [Literary Universe](https://www.literaryuniverse.com)
I'm not a designer so it doesn't look much. Also I have been implementing Reakit since the early days of 1.0 beta, so I still have to go back and redo some of the older components.

**Comment by btzr-io** [comment]
I'm using reakit for the toolbar component in my new project: [Villain](https://github.com/btzr-io/Villain), an open source web-based comic book reader. Currently is just a react component but the idea is to have a web app + embed api. 

This comic book viewer is also used inside the [`lbry-desktiop`](https://github.com/lbryio/lbry-desktop) project.

There is no website yet, but I made a logo although I need to update the image :stuck_out_tongue:

**Comment by folz** [comment]
We’re using reakit and reakit-system at @Nextdoor to build our design system’s component library. You can use our logo to indicate we use reakit, it's at https://about.nextdoor.com/us-media/.

Reakit’s first-principles focus on accessibility is a great foundation to build on top of, and we like reakit-system’s unique approach to behavior composition. All of our component library is implemented using reakit-system in a similar way to how reakit itself is built.

Since our component library is a new project, its usage is relatively low compared to our existing, non-reakit components. Even so, we already use reakit-powered components in core parts of our product, and we’ve been able to replace some of our primitive UI components with reakit-powered implementations.

**Comment by Grsmto** [comment]
You probably know already but ThemeUI (and by extension, the Gatsby team) [is now using Reakit](https://github.com/system-ui/theme-ui/blob/5f36f7cc6753e02bab2192f36d5bd67bedc21e34/packages/editor/package.json#L21). 👍

**Comment by diegohaz** [comment]
Reakit [is being used by WordPress](https://github.com/WordPress/gutenberg/blob/4857ad58c1241b3d63d21a6880c989b85746c3dc/packages/components/src/toolbar/toolbar-container.js#L4) and other projects at @Automattic.

**Comment by gziolo** [comment]
> Reakit [is being used by WordPress](https://github.com/WordPress/gutenberg/blob/4857ad58c1241b3d63d21a6880c989b85746c3dc/packages/components/src/toolbar/toolbar-container.js#L4) and other projects at @Automattic.

We want to expand the usage of Reakit in the WordPress components npm package 😃 In fact, Reakit is now part of WordPress core and is bundled in the official distribution. Thanks @diegohaz for all the work to make it happen!

**Comment by christiankaindl** [comment]
Hi there, we at @quantargo are using Reakit for our website and online course platform (free coding courses). Currently we are using Tooltips, Popovers, Dialogs, Checkboxes and Radios - And we will keep migrating more components to improve a11y of our site. I gotta say, we very much enjoy the API decisions the Reakit team has made! Really, big thanks ❤️ 

Our site is https://quantargo.com, feel free to include us wherever you see fit :)

**Comment by TheSisb** [comment]
[Twilio](https://twilio.com)'s [Paste design system](https://paste.twilio.design) is using Reakit. It has been excellent to work with so far. Our [repository](https://github.com/twilio-labs/paste) is open source.  

Thank you for an excellent library! <3

**Comment by tolunayakbulut** [comment]
Hey there, we are using Reakit on [radity.com](https://radity.com/). Thanks for developing Reakit.

**Comment by MirzayevFarid** [comment]
We’re using Reakit on [globalctoforum.org](https://globalctoforum.org/) in many of our new components.

Thanks for such an amazing work!
You can add us to the list.

**Comment by frassinier** [comment]
Reakit is proudly used for [Talend](https://talend.com) [design system](https://github.com/Talend/design-system)

**Comment by Miaxos** [comment]
We're using Reakit on [iAdvize.com](https://iadvize.com) and we love it 

Thank you for this amazing work !

**Comment by halukugurlu** [comment]
We are using it at [LimeLight](https://www.mylimelight.com/).

Thanks a lot for your amazing work!

**Comment by gnapse** [comment]
We're starting to use Reakit at [Doist](https://doist.com) to implement some of our own more opinionated components on top of it. This includes work in both of our products [Todoist](https://todoist.com) and [Twist](https://twist.com).

The experience has been great so far. We're really loving that Reakit sometimes is a bit low level, as it allows us to really customize it. The new composite components are amazing. Also great for us is the design decision to have consumers being in charge of passing hook results down as props. Sometimes it feels too much work for the consumer, but it allows us to customize stuff even more. We tend to then gain back the magic by doing the pass down via a context.

Thanks a lot for all this!

**Comment by tommoor** [comment]
We just [started using Reakit](https://github.com/outline/outline/pull/1768) at [Outline](https://www.getoutline.com/).

I was able to completely remove a custom menu system that had been built over the years, but it wasn't accessible at all.  ~Hoping to convert our modals next~ 👍 

Edit: Modals converted too!

**Comment by Liinkiing** [comment]
I'm using Reakit for a lot of my components for a handmade UI kit made for learning purposes! Thanks to you it handles a lot of complexe use cases with a11y and I combine it with framer-motion for smooth transitions for my Modals / Drawers / Menu! 

If you wanna take a look, it's open source! 👍 https://github.com/Liinkiing/klee (again, thank you for this library, the API is really well thought and handles all the hard stuff but it is mandatory because a11y is mandatory and should not be a feature!)

**Comment by P1X3L** [comment]
👋 Here at [Welcome To The Jungle](https://www.welcometothejungle.com), we use (and love) `reakit` on [www.welcome-ui](https://welcome-ui.com), our beloved open sourced design system.

We would be thrilled to have our logo on your site or README!

Thank you for all the great work!

**Comment by navin-moorthy** [comment]
👋 We at [Timeless](https://twitter.com/timelessco) believe in the [idea of Layered architectures](https://surma.dev/things/cost-of-convenience/) which will help us build amazing products with less frustration.

Ariakit is one of the library that provides this kind of API which makes easy to customize and build products on top of the primtives.

We use Ariakit in our Design System
- Repo: https://github.com/adaptui/react-tailwind
- Storybook: https://adaptui-react-tailwind.vercel.app/

We also extend some Ariakit components for our usage in a separate library which might be added to Ariakit in future,
- Repo: https://github.com/adaptui/react
- Storybook: https://adaptui-react.vercel.app/
